tests/ballerina-integration-test/src/test/resources/websub/services/01_websub_publisher.bal

```diff
@@ -227,3 +230,26 @@ function getPayloadContent(string contentType, string mode) returns string|xml|j
     error e = error(websub:WEBSUB_ERROR_CODE, { message : errorMessage });
     panic e;
 }
+
+function checkSubscriberAvailability(string topic, string callback) {
+    boolean subscriberAvailable = false;
+    while(!subscriberAvailable) {
+        websub:SubscriberDetails[] topicDetails = webSubHub.getSubscribers(topic);
+        if (isSubscriberAvailable(topicDetails, callback)) {
+            subscriberAvailable = true;
```

Comment:
can we return here and remove the else block to have just `runtime:sleep(1000);`?